internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -1574,6 +1574,7 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 				string(managedclusters.UpgradeChannelStable),
 				string(managedclusters.UpgradeChannelNodeNegativeimage),
 			}, false),
+			Deprecated: features.DeprecatedInFourPointOh("The property `automatic_channel_upgrade` will be renamed to `automatic_upgrade_channel` in v4.0"),
```

Comment:
_minor_ for consistency with the others:

```suggestion
			Deprecated: features.DeprecatedInFourPointOh("The property `automatic_channel_upgrade` will be renamed to `automatic_upgrade_channel` in v4.0 of the AzureRM Provider"),
```